### DISCUSSION TITLE
Steuer-Kommandos zur opendtu // Projekt Nulleinspeisung

### DISCUSSION THREAD

**Discussion by Al-Sol-We** [tbnobody/OpenDTU#2111] [Q&A]
Hallo,
 ich bin neu in diesem Forum. Interessieren würde mich ob, es nicht eine Auflistung von Steuerkommandos für die openDTU gibt. Ich habe ein Netzwerk bereits aufgebaut und nutze Shelly 3EM und kann mit einem Raspberry PiCo-W bereits die Shelly 3M auslesen und entscheiden, ob Energie aus einem zweiten Solar_Netzwerk ins Haus eingespeist werden soll oder nicht. Es wäre für mich interessant, zu erfahren welches Kommando zum Beispiel das Limit einstellt  (nicht permanentes Limit).
Es werden im Handel auch OpenDTUs angeboten, die bereits die Nulleinspeisungfunktion beinhalten. Ich wundere mich ein wenig, dass es  zusätzlich zur OpenDTU auch noch das Projekt mit diesem Solarmanager „Smart-Micro-Solar“  (für zusätzlich 129€) gibt. Wo sind dort die Unterschiede, bzw. wer kauf denn die 2 Komponenten, wenn mal alles in einem Gerät bekommen kann?

Viele Grüße

**Comment by Gururom** [comment]
SmartMicroSolar ist halt ein kmmerzielles Produkt, kannst openDTUonBattery nutzen, da ist eine Reglung für die Hoymiles WR inkl.
